tests/machine/hera.ncar

```diff
@@ -0,0 +1,32 @@
+  source bl_date.ncar.conf
+  INPUT_DATE=20230920
+
+  module load rocoto
+  ROCOTORUN=$(which rocotorun)
+  ROCOTOSTAT=$(which rocotostat)
+  ROCOTOCOMPLETE=$(which rocotocomplete)
+  ROCOTO_SCHEDULER=slurm
+
+  PYTHONHOME=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3_new_20210629
+  export PATH=$PYTHONHOME/bin:$PATH
+  export PYTHONPATH=$PYTHONHOME/lib/python3.7/site-packages
+
+  module load ecflow
+  ECFLOW_START=ecflow_start.sh
+
+  QUEUE=batch
+  COMPILE_QUEUE=batch
+
+  PARTITION=
+  dprefix=${RUNDIR_ROOT:-/scratch1/BMC/gmtb/CCPP_regression_testing/NCAR_ufs-weather-model}
+  DISKNM=$dprefix/
+  STMP=$dprefix/stmp4
+  PTMP=$dprefix/stmp2
+  RTPWD=${RTPWD:-/scratch1/BMC/gmtb/CCPP_regression_testing/NCAR_ufs-weather-model/baselines/main-${BL_DATE}/}
```

Comment:
Who has permissions in here? I'd like to be able to clear this out when necessary.

Reply:
Same comment for all directories that we'll be writing to. Everyone tasked with running RTs should have write permissions in the directories in order to clear space.